server/src/main/java/org/opensearch/snapshots/RestoreService.java

```diff
@@ -165,8 +165,8 @@ public class RestoreService implements ClusterStateApplier {
             SETTING_CREATION_DATE,
             SETTING_HISTORY_UUID,
             SETTING_REMOTE_STORE_ENABLED,
-            SETTING_REMOTE_SEGMENT_STORE_REPOSITORY,
-            SETTING_REMOTE_TRANSLOG_STORE_REPOSITORY
+            SETTING_REMOTE_SEGMENT_STORE_DATA_REPOSITORY,
+            SETTING_REMOTE_TRANSLOG_STORE_DATA_REPOSITORY
```

Comment:
do we need to add the metadata repo as well to unmodified settings during snapshot restore?

Reply:
+1. We will need corresponding ITs as well to restore .